Cargo.toml

```diff
@@ -37,6 +37,8 @@ home = "0.5.9"
 itertools = "0.13.0"
 pad-adapter = "0.1.1"
 serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+display_json = "=0.2.1"
```

Comment:
This dependency is using `syn1`, so I prefer to avoid it. Also, the part we are using is easy to replicate. Just add the following to `crates/flux-macros/src/lib.rs`


```rust
use quote::quote;

...

decl_derive!(
    [DebugAsJson] => debug_as_json
);

fn debug_as_json(s: synstructure::Structure<'_>) -> proc_macro2::TokenStream {
    s.gen_impl(quote! {
        gen impl std::fmt::Debug for @Self {
            fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
                let s = serde_json::to_string(&self).map_err(|_| std::fmt::Error::default())?;
                write!(f, "{}", s)
            }
        }
    })
}
```

Reply:
To be clear, you mean remove the dependency on `display_json` but keep `serde_json` yes? (because we already depend on the latter in fixpoint, and also in the above snippet?)